editor/src/components/canvas/canvas-strategies/strategies/draw-to-insert-metastrategy.tsx

```diff
@@ -295,47 +302,74 @@ function getHighlightAndReorderIndicatorCommands(
 function getInsertionCommands(
   subject: InsertionSubject,
   interactionSession: InteractionSession,
-  insertionSubjectSize: Size,
+  insertionSubjectSize: Size | 'skip-size-props',
   sizing: 'zero-size' | 'default-size',
 ): { command: InsertElementInsertionSubject; frame: CanvasRectangle } | null {
   if (
     interactionSession.interactionData.type === 'DRAG' &&
     (sizing === 'default-size' || interactionSession.interactionData.drag != null)
   ) {
     const pointOnCanvas = interactionSession.interactionData.dragStart
+    if (insertionSubjectSize === 'skip-size-props') {
+      const frame = canvasRectangle({
```

Comment:
It looks like this side of the condition is running the exact same code as if `sizing === 'zero-size'` below